presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -61,3 +61,9 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+
+.. function:: crc32(binary) -> bigint
```

Comment:
(there are enough functions here to warrant separate sections, but that should be fixed separately)